lib/puppet/provider/netscaler_sslservice/rest.rb

```diff
@@ -0,0 +1,69 @@
+require 'puppet/provider/netscaler'
+require 'json'
+
+Puppet::Type.type(:netscaler_sslservice).provide(:rest, {:parent => Puppet::Provider::Netscaler}) do
+	def self.instances
+		instances = []
+		sslservices = Puppet::Provider::Netscaler.call('/config/sslservice')
+		return [] if sslservices.nil?
+
+		sslservices.each do |sslservice|
+			instances << new({
+				:ensure              => :present,
+				:name                => sslservice['servicename'],
+				:cipherdetails       => sslservice['cipherdetails'],
```

Comment:
cipher_details is more apt following what is in https://github.com/puppetlabs/puppetlabs-netscaler/blob/master/lib/puppet/provider/netscaler_csaction/rest.rb#L18-L19